Makefile

```diff
@@ -12,3 +12,14 @@ include common/common.mk
 # Additional upgrade tests.  Not hooked into CI ATM.
 upgrade-tests: $(VERSIONS)
 	test/run_upgrade_test 9.2:remote 9.4:local 9.5:local 9.6:local
+
+DG = /usr/bin/dg
+DG_EXEC = ${DG} --max-passes 25 --template Dockerfile.template --multispec specs/multispec.yml
+
+generate_dockerfiles:
+	${DG_EXEC} --distro centos-7-x86_64.yaml --multispec-selector version="9.4" --output 9.4/Dockerfile
+	${DG_EXEC} --distro centos-7-x86_64.yaml --multispec-selector version="9.5" --output 9.5/Dockerfile
+	${DG_EXEC} --distro centos-7-x86_64.yaml --multispec-selector version="9.6" --output 9.6/Dockerfile
+	${DG_EXEC} --distro rhel-7-x86_64.yaml --multispec-selector version="9.4" --output 9.4/Dockerfile.rhel7
+	${DG_EXEC} --distro rhel-7-x86_64.yaml --multispec-selector version="9.5" --output 9.5/Dockerfile.rhel7
+	${DG_EXEC} --distro rhel-7-x86_64.yaml --multispec-selector version="9.6" --output 9.6/Dockerfile.rhel7
```

Comment:
@dhodovsk wrote (dunno where the comment disappeared):
> You mean one of [these](https://github.com/devexp-db/distgen/tree/master/distconf) or would you like to see a different approach for obtaining distribution specific information?

You switched from rhel-7.3-x86_64 to rhel-7-x86_64, so we don't have to bump the version all the time, looks better.

Though here I would go with `for version in $(VERSIONS); do .... done` loop.

Reply:
Ping on this.  I often do 'make VERSIONS=9.6' only, for this target it would be useful, too.